packages/e2e-tests/test-applications/create-next-app/tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+  "compilerOptions": {
+    "target": "es5",
+    "lib": ["dom", "dom.iterable", "esnext"],
+    "allowJs": true,
+    "skipLibCheck": true,
+    "strict": true,
+    "forceConsistentCasingInFileNames": true,
+    "noEmit": true,
+    "esModuleInterop": true,
+    "module": "esnext",
+    "moduleResolution": "node",
+    "resolveJsonModule": true,
+    "isolatedModules": true,
+    "jsx": "preserve",
+    "incremental": true
```

Comment:
I am getting an error "Option 'noEmit' cannot be specified with option 'incremental'." here. Is this something you added or something Next.js generated?

Reply:
Yes, that's auto-generated / modified by Next.JS. Tried to suppress it, but thought that might make the application diverted from the original `create-next-app`. This will get resolved when we update our TS version, though.